mindmap/mindmap.py

```diff
@@ -148,7 +196,9 @@ def studio_view(self, context=None) -> Fragment:
         # Add i18n js
         statici18n_js_url = self._get_statici18n_js_url()
         if statici18n_js_url:
-            frag.add_javascript_url(self.runtime.local_resource_url(self, statici18n_js_url))
+            frag.add_javascript_url(
+                self.runtime.local_resource_url(self, statici18n_js_url)
+            )
 
         frag.add_javascript(self.resource_string("static/js/src/mindmapEdit.js"))
```

Comment:
[courious] why don't we add `frag.add_javascript(self.resource_string("static/js/src/requiredModules.js"))` here?

Reply:
We tested adding the js file, but it still doesn't work.